src/main/java/io/github/moulberry/notenoughupdates/profileviewer/DungeonPage.java

```diff
@@ -63,7 +63,7 @@ public class DungeonPage extends GuiProfileViewerPage {
 		new ItemStack(Items.bow),
 		new ItemStack(Items.leather_chestplate),
 	};
-	private static final String[] bossFloorArr = { "Bonzo", "Scarf", "Professor", "Thorn", "Livid", "Sadan", "Necron" };
+	private static final String[] bossFloorArr = {"Bonzo", "Scarf", "Professor", "Thorn", "Livid", "Sadan", "Necron"};
```

Comment:
We surely have this data already somewhere else saved, yes?

Reply:
No why would we?

Reply:
Makes no sense to have them somewhere else as we only use them there.